Offline/OfflinePlayer/objc/OfflinePlayer/ManagersUtilities/DownloadManager.m

```diff
@@ -259,6 +261,13 @@ - (void)downloadVideoFromQueue
     __weak typeof(self) weakSelf = self;
     
     AVURLAsset *avURLAsset = [self.offlineVideoManager urlAssetForVideo:video error:nil];
+    
+    if ([avURLAsset.URL.absoluteString containsString:@"aes128"])
+    {
+        [UIAlertController showAlertWithTitle:@"Content Not Supported" message:@"Offline playback is not supported for HLSe content." actionTitle:@"OK" inController:InterfaceManager.sharedInstance.tabBarController];
```

Comment:
Question: how many other showAlert's are there in the SDK?

Reply:
This isn't in the SDK itself, it's just a convenience method from a class in this project. The Swift version has a similar one.

Reply:
ah. sorry I didn't look closely enough. this is fine. we're backed up by https://sdks.support.brightcove.com/features/offline-playback-native-player-sdks.html (search for HLSe). but it is still a nuisance that you spent that much time tracking down a user error.